<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="1" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-02-17T08:58:23 IST" name="Suite" finished-at="2023-02-17T08:58:55 IST" duration-ms="32045">
    <groups>
    </groups>
    <test started-at="2023-02-17T08:58:23 IST" name="Test" finished-at="2023-02-17T08:58:55 IST" duration-ms="32045">
      <class name="TestCases.verifyLogin">
        <test-method is-config="true" signature="openurl()[pri:0, instance:TestCases.verifyLogin@4b8ee4de]" started-at="2023-02-17T08:58:23 IST" name="openurl" finished-at="2023-02-17T08:58:31 IST" duration-ms="7796" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openurl -->
        <test-method signature="login(java.lang.String,java.lang.String)[pri:0, instance:TestCases.verifyLogin@4b8ee4de]" started-at="2023-02-17T08:58:31 IST" name="login" data-provider="testData" finished-at="2023-02-17T08:58:31 IST" duration-ms="716" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[testuser1]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secure123]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
Error message is not matching expected [Please check your and password. If you still can't log in, contact your Salesforce administrator., Please check your and password. If you still can't log in, contact your Salesforce administrator.] but found [Please check your username and password. If you still can't log in, contact your Salesforce administrator.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
Error message is not matching expected [Please check your and password. If you still can't log in, contact your Salesforce administrator., Please check your and password. If you still can't log in, contact your Salesforce administrator.] but found [Please check your username and password. If you still can't log in, contact your Salesforce administrator.]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at Resources.commonMethods.handleAssertions(commonMethods.java:26)
at TestCases.verifyLogin.login(verifyLogin.java:29)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:TestCases.verifyLogin@4b8ee4de]" started-at="2023-02-17T08:58:31 IST" name="closeBrowser" finished-at="2023-02-17T08:58:32 IST" duration-ms="785" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method is-config="true" signature="openurl()[pri:0, instance:TestCases.verifyLogin@4b8ee4de]" started-at="2023-02-17T08:58:32 IST" name="openurl" finished-at="2023-02-17T08:58:37 IST" duration-ms="5025" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openurl -->
        <test-method signature="login(java.lang.String,java.lang.String)[pri:0, instance:TestCases.verifyLogin@4b8ee4de]" started-at="2023-02-17T08:58:37 IST" name="login" data-provider="testData" finished-at="2023-02-17T08:58:38 IST" duration-ms="1023" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[testuser2]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[secure456]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
Error message is not matching expected [Please check your and password. If you still can't log in, contact your Salesforce administrator., Please check your and password. If you still can't log in, contact your Salesforce administrator.] but found [Please check your username and password. If you still can't log in, contact your Salesforce administrator.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
Error message is not matching expected [Please check your and password. If you still can't log in, contact your Salesforce administrator., Please check your and password. If you still can't log in, contact your Salesforce administrator.] but found [Please check your username and password. If you still can't log in, contact your Salesforce administrator.]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at Resources.commonMethods.handleAssertions(commonMethods.java:26)
at TestCases.verifyLogin.login(verifyLogin.java:29)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:TestCases.verifyLogin@4b8ee4de]" started-at="2023-02-17T08:58:38 IST" name="closeBrowser" finished-at="2023-02-17T08:58:39 IST" duration-ms="820" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- TestCases.verifyLogin -->
      <class name="TestCases.verifySignup">
        <test-method is-config="true" signature="openurl()[pri:0, instance:TestCases.verifySignup@4145bad8]" started-at="2023-02-17T08:58:39 IST" name="openurl" finished-at="2023-02-17T08:58:43 IST" duration-ms="4087" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openurl -->
        <test-method signature="signup()[pri:0, instance:TestCases.verifySignup@4145bad8]" started-at="2023-02-17T08:58:43 IST" name="signup" finished-at="2023-02-17T08:58:54 IST" duration-ms="10880" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- signup -->
        <test-method is-config="true" signature="closeBrowser()[pri:0, instance:TestCases.verifySignup@4145bad8]" started-at="2023-02-17T08:58:54 IST" name="closeBrowser" finished-at="2023-02-17T08:58:55 IST" duration-ms="780" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- TestCases.verifySignup -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
